openmls/src/framing/ciphertext.rs

```diff
@@ -33,9 +34,8 @@ impl MlsCiphertext {
         mls_plaintext: &MlsPlaintext,
```

Comment:
I had the same reflex, but I didn't do it because I thought it would blur the separation. I Don't have a very strong opinion, but changing it would probably mean it's not easy to test without having an actual group.

Reply:
Maybe an intermediate struct that can readily be created from a reference to a group?

Reply:
Yes that's the downside. I'm fine with keeping it this way since you got rid of the `RefCell` anyway. Otherwise that would've been the solution.